components/integration/code/integration-bbe/unleash-the-power-of-ai.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Unleash the power of AI'
-description: "With Ballerina's AI connectors, AI libraries, and built-in language and standard library features, incorporating AI into your integrations can be done in a matter of seconds, enabling you to quickly build intelligent and efficient workflows."
+description: "With Ballerina's AI connectors, AI libraries, and built-in language and Ballerina library features, incorporating AI into your integrations can be done in a matter of seconds, enabling you to quickly build intelligent and efficient workflows."
```

Comment:
This need full rewrite as AI connectors, AI libraries (incorrect to say libraries here and should use packages) and same comment related to built-in language above applies here too.

Reply:
Changed to `language and library features of Ballerina`.